test/integrations/destinations/google_adwords_offline_conversions/dataDelivery/business.ts

```diff
@@ -0,0 +1,728 @@
+
+export const testScenariosForV0API = [
+  {
+    id: 'gaoc_v0_scenario_1',
+    name: 'google_adwords_offline_conversions',
+    description:
+      '[Proxy v0 API] :: Test for invalid argument - where the destination responds with 400 with invalid argument error',
+    successCriteria: 'Should return 400 with error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          headers: headers.header1,
+          params: params.param1,
+          JSON: invalidArgumentRequestPayload,
+          endpoint:
+            'https://googleads.googleapis.com/v14/customers/11122233331/offlineUserDataJobs',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 400,
+        body: {
+          output: {
+            status: 400,
+            message:
+              '[Google Ads Offline Conversions]:: Request contains an invalid argument. during google_ads_offline_store_conversions Add Conversion',
+            destinationResponse: {
+              error: {
+                code: 400,
+                details: [
+                  {
+                    '@type': 'type.googleapis.com/google.ads.googleads.v14.errors.GoogleAdsFailure',
+                    errors: [
+                      {
+                        errorCode: {
+                          offlineUserDataJobError: 'INVALID_SHA256_FORMAT',
+                        },
+                        message: 'The SHA256 encoded value is malformed.',
+                        location: {
+                          fieldPathElements: [
+                            {
+                              fieldName: 'operations',
+                              index: 0,
+                            },
+                            {
+                              fieldName: 'create',
+                            },
+                            {
+                              fieldName: 'user_identifiers',
+                              index: 0,
+                            },
+                            {
+                              fieldName: 'hashed_email',
+                            },
+                          ],
+                        },
+                      },
+                    ],
+                    requestId: '68697987',
+                  },
+                ],
+                message: 'Request contains an invalid argument.',
+                status: 'INVALID_ARGUMENT',
+              },
+            },
+            statTags: {
+              destType: 'GOOGLE_ADWORDS_OFFLINE_CONVERSIONS',
+              destinationId: 'default-destinationId',
+              errorCategory: 'network',
+              errorType: 'aborted',
+              feature: 'dataDelivery',
+              implementation: 'native',
+              module: 'destination',
+              workspaceId: 'default-workspaceId',
```

Comment:
Can this be deduplicated by creating a constant variable

Reply:
addressed